src/libs/actions/QueuedOnyxUpdates.ts

```diff
@@ -1,30 +1,26 @@
-import Onyx from 'react-native-onyx';
+import Onyx, {OnyxUpdate} from 'react-native-onyx';
 import ONYXKEYS from '../../ONYXKEYS';
 
 // In this file we manage a queue of Onyx updates while the SequentialQueue is processing. There are functions to get the updates and clear the queue after saving the updates in Onyx.
 
-let queuedOnyxUpdates = [];
+let queuedOnyxUpdates: OnyxUpdate[] = [];
 Onyx.connect({
     key: ONYXKEYS.QUEUED_ONYX_UPDATES,
-    callback: (val) => (queuedOnyxUpdates = val || []),
+    callback: (val) => (queuedOnyxUpdates = val ?? []),
 });
 
 /**
- * @param {Array<Object>} updates Onyx updates to queue for later
- * @returns {Promise}
+ * @param updates Onyx updates to queue for later
  */
-function queueOnyxUpdates(updates) {
+function queueOnyxUpdates(updates: OnyxUpdate[]): Promise<void> {
     return Onyx.set(ONYXKEYS.QUEUED_ONYX_UPDATES, [...queuedOnyxUpdates, ...updates]);
 }
 
-function clear() {
+function clear(): void {
```

Comment:
```suggestion
function clear() {
```

I think we don't need to type `void`